hack/generate/gen-test-framework.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+#############################
+# Make sure that test/test-framework is updated according to the current source code
+#############################
+
+set -ex
+
+source hack/lib/test_lib.sh
+export GO111MODULE=on
+
+# Define vars
+DOCKERFILE="build/Dockerfile"
+GOMOD="go.mod"
+
+# Go inside of the mock data test project
+cd test/test-framework
+
+# Create the required files in the mock test/test-framework just to allow run from it the oprator-sdk commands
+echo > $DOCKERFILE
+echo > $GOMOD
+
+# Remove files that are required and created just to exec the commands.
+trap_add 'rm -rf $DOCKERFILE' EXIT
+trap_add 'rm -rf $GOMO' EXIT
+trap_add 'rm -rf go.sum' EXIT
+
+# Run gen commands
+operator-sdk generate k8s
+operator-sdk generate openapi
+
+
+
+
+
```

Comment:
Nit: Extraneous newlines at the end of the file. The last line should have a trailing new line, but then no further new lines after that.